cpp/log/etcd_consistent_store.h

```diff
@@ -106,12 +103,12 @@ class EtcdConsistentStore : public ConsistentStore<Logged> {
 
   std::string GetUnsequencedPath(const std::string& hash) const;
 
-  std::string GetSequencedPath(int64_t seq) const;
-
   std::string GetNodePath(const std::string& node_id) const;
 
   std::string GetFullPath(const std::string& key) const;
 
+  void CheckMappingIsOrdered(const ct::SequenceMapping& mapping) const;
```

Comment:
Doesn't really need to be a member, could be a plain function in the anonymous namespace?